data-structures-stacks/data-structures-stacks-quiz-notes.md

```diff
@@ -0,0 +1,41 @@
+# data-structures-stacks-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What does the acronym LIFO mean?
+  LIFO stands for last in, first out.
+- What methods are available on a Stack data structure?
+  push(value), pop(), peek()
+- What must you do to access the value at an arbitrary point in a stack (not just the "top")?
+  You must create a helper function that pops the top and grabs it. After accessing it, we need to
+  push the values back onto the stack.
```

Comment:
While that may work for the value just beneath the top, how would you access a value _arbitrarily deep_ within the stack?